dind-aws/Dockerfile

```diff
@@ -17,6 +19,24 @@ RUN echo "Install AWS Cli" && \
     chmod 755 /usr/bin/ci-helper && \
     echo "Done installing CI Helper" && \
 
+    echo "Update Docker" && \
+    apt-get -y remove docker docker-engine docker.io && \
+    apt-get -qy install apt-transport-https ca-certificates curl software-properties-common && \
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | apt-key add - && \
+    add-apt-repository \
+        "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+        $(lsb_release -cs) \
+        stable" && \
+    apt-get -q update && \
+    apt-get -qy install docker-ce=${DOCKER_VERSION} && \
+    docker --version && \
+
+    echo "Update Docker Compose" && \
+    rm -f /usr/local/bin/docker-compose && \
+    curl -L https://github.com/docker/compose/releases/download/${DOCKER_COMPOSE_VERSION}/docker-compose-`uname -s`-`uname -m` -o /usr/local/bin/docker-compose && \
```

Comment:
`curl -sSL` would be better than just `curl -L` : silent, yet errors are reported.

Reply:
done